apps/web/playwright/organization/booking.e2e.ts

```diff
@@ -28,9 +29,9 @@ function getOrgOrigin(orgSlug: string | null) {
 }
 
 test.describe("Bookings", () => {
-  test.afterEach(({ orgs, users }) => {
-    orgs.deleteAll();
-    users.deleteAll();
+  test.afterEach(async ({ orgs, users, page }) => {
+    await users.deleteAll();
```

Comment:
It is important to delete users first now. Because if we delete orgs first we endup with a scenario where two different users with same username exist outside org which is not possible and query errors.